lib/tasks/generate_2fa_encription_key.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+namespace :generate do
+  desc "Generates a key for encrypting 2fa tokens"
+  task :twofa_key do
+    path = Rails.root.join("config", "initializers", "twofa_encryption_key.rb")
+    key = SecureRandom.hex(32)
+    File.open(path, "w") do |f|
+      f.write <<EOF
+# The 2fa encryption key is used to encrypt users' OTP tokens in the database.
```

Comment:
Layout/IndentHeredoc: Use 2 spaces for indentation in a heredoc by using `<<~` instead of `<<`.